GameData/SPEngine/Parts/LR87_proc.cfg

```diff
@@ -25,3 +25,31 @@
 		@name = Hypergolic-Lower
 	}
 }
+
++PART[ROE-LR87]:NEEDS[ROEngines]:BEFORE[ZREALPLUME]
```

Comment:
I don't think this'll work: the ROE-LR87 is (I presume) a twin-chamber model (i.e. a full LR87) whereas the T-class engine is a single-chamber engine (half an LR87).

Reply:
Might it be added as an alternative model instead?

Reply:
I guess we could, but it'll be severely under-thrusted for its size, so we'll want to scale it down (but isotropically, not squeezed thin like the current model).

Reply:
I've made this fix locally on top of your 11af5ba (I also rebased on top of some of my local changes which have a minor conflict) and am about to test.